easybuild/easyconfigs/e/Evcxr-REPL/Evcxr-REPL-0.14.2-GCCcore-12.2.0-Rust-1.65.0.eb

```
@@ -0,0 +1,456 @@
+    {'unicase-2.6.0.tar.gz': '50f37be617794602aabbeee0be4f259dc1778fabe05e2d67ee8f79326d5cb4f6'},
+    {'unicode-bidi-0.3.8.tar.gz': '099b7128301d285f79ddd55b9a83d5e6b9e97c92e0ea0daebee7263e932de992'},
+    {'unicode-ident-1.0.6.tar.gz': '84a22b9f218b40614adcb3f4ff08b703773ad44fa9423e4e0d346d5db86e4ebc'},
+    {'unicode-normalization-0.1.22.tar.gz': '5c5713f0fc4b5db668a2ac63cdb7bb4469d8c9fed047b1d0292cc7b0ce2ba921'},
+    {'unicode-segmentation-1.10.0.tar.gz': '0fdbf052a0783de01e944a6ce7a8cb939e295b1e7be835a1112c3b9a7f047a5a'},
+    {'unicode-width-0.1.10.tar.gz': 'c0edd1e5b14653f783770bce4a4dabb4a5108a5370a5f5d8cfe8710c361f6c8b'},
+    {'unicode-xid-0.2.4.tar.gz': 'f962df74c8c05a667b5ee8bcf162993134c104e96440b663c8daa176dc772d8c'},
+    {'url-2.3.1.tar.gz': '0d68c799ae75762b8c3fe375feb6600ef5602c883c5d21eb51c09f22b83c4643'},
+    {'utf8parse-0.2.0.tar.gz': '936e4b492acfd135421d8dca4b1aa80a7bfc26e702ef3af710e0752684df5372'},
+    {'vec_map-0.8.2.tar.gz': 'f1bddf1187be692e79c5ffeab891132dfb0f236ed36a43c7ed39f1165ee20191'},
+    {'version_check-0.9.4.tar.gz': '49874b5167b65d7193b8aba1567f5c7d93d001cafc34600cee003eda787e483f'},
+    {'walkdir-2.3.2.tar.gz': '808cf2735cd4b6866113f648b791c6adc5714537bc222d9347bb203386ffda56'},
+    {'wasi-0.11.0+wasi-snapshot-preview1.tar.gz': '9c8d87e72b64a3b4db28d11ce29237c246188f4f51057d65a7eab63b7987e423'},
+    {'which-4.3.0.tar.gz': '1c831fbbee9e129a8cf93e7747a82da9d95ba8e16621cae60ec2cdc849bacb7b'},
+    {'winapi-0.3.9.tar.gz': '5c839a674fcd7a98952e593242ea400abe93992746761e38641405d28b00f419'},
+    {'winapi-i686-pc-windows-gnu-0.4.0.tar.gz': 'ac3b87c63620426dd9b991e5ce0329eff545bccbbb34f3be09ff6fb6ab51b7b6'},
+    {'winapi-util-0.1.5.tar.gz': '70ec6ce85bb158151cae5e5c87f95a8e97d2c0c4b001223f33a334e3ce5de178'},
+    {'winapi-x86_64-pc-windows-gnu-0.4.0.tar.gz': '712e227841d057c1ee1cd2fb22fa7e5a5461ae8e48fa2ca79ec42cfc1931183f'},
+    {'windows-sys-0.42.0.tar.gz': '5a3e1820f08b8513f676f7ab6c1f99ff312fb97b553d30ff4dd86f9f15728aa7'},
+    {'windows_aarch64_gnullvm-0.42.0.tar.gz': '41d2aa71f6f0cbe00ae5167d90ef3cfe66527d6f613ca78ac8024c3ccab9a19e'},
+    {'windows_aarch64_msvc-0.42.0.tar.gz': 'dd0f252f5a35cac83d6311b2e795981f5ee6e67eb1f9a7f64eb4500fbc4dcdb4'},
+    {'windows_i686_gnu-0.42.0.tar.gz': 'fbeae19f6716841636c28d695375df17562ca208b2b7d0dc47635a50ae6c5de7'},
+    {'windows_i686_msvc-0.42.0.tar.gz': '84c12f65daa39dd2babe6e442988fc329d6243fdce47d7d2d155b8d874862246'},
+    {'windows_x86_64_gnu-0.42.0.tar.gz': 'bf7b1b21b5362cbc318f686150e5bcea75ecedc74dd157d874d754a2ca44b0ed'},
+    {'windows_x86_64_gnullvm-0.42.0.tar.gz': '09d525d2ba30eeb3297665bd434a54297e4170c7f1a44cad4ef58095b4cd2028'},
+    {'windows_x86_64_msvc-0.42.0.tar.gz': 'f40009d85759725a34da6d89a94e63d7bdc50a862acf0dbc7c8e488f1edcb6f5'},
+    {'yansi-0.5.1.tar.gz': '09041cd90cf85f7f8b2df60c646f853b7f535ce68f85244eb6731cf89fa498ec'},
+]
+
+dependencies = [
```

Since we're using `GCCcore` as toolchain, `binutils` should be a build dep:

```suggestion
builddependencies = [
    ('binutils', '2.39'),
]

dependencies = [